### ISSUE TITLE
Mutations with authorization header gives 500 error

### ISSUE THREAD

**Issue by vorasudh** [wp-graphql/wp-graphql-woocommerce#660]
**Describe the bug**
I am facing issues when a logged-in customer is trying to checkout. I am using WPGraphQL JWT Authentication for authentication. I receive `authToken`, `refreshToken` and `sessionToken`. When I try to make any mutation with the `authorization` and `woocommerce-session` header, the call fails with error code 500 with this message:

`<p>There has been a critical error on this website.</p><p><a href=\"https://wordpress.org/support/article/faq-troubleshooting/\">Learn more about troubleshooting WordPress.</a></p>`.

Unfortunately, enabling debugging does not provide any logs in response to the mutation.

So basically a logged-in user cannot add a product to a cart (mutation: addToCart) and cannot checkout (mutation: checkout). I have looked at many previous issues (#88, #147, #297, #320), but nothing is solving my problem here. Maybe I missed some configuration here. Can someone please guide me?

**Plugin Versions**
- **WooGraphQL Version:** 0.11.2
- **WPGraphQL Version:** 1.11.1
- **WordPress Version:** 6.0.2
- **WooCommerce Version:** 6.9.4

**Comment by kidunot89** [comment]
@vorasudh What version of **WPGraphQL JWT Authentication** are you using?

**Comment by vorasudh** [comment]
@kidunot89 I am using v0.5.2 of **WPGraphQL JWT Authentication**

**Comment by kidunot89** [comment]
@vorasudh Are you testing this in a GraphQL client?

**Comment by vorasudh** [comment]
Nopes, i am using Apollo client in my NextJS project.
